packages/generator-emakinacee-react/generators/app/index.js

```diff
@@ -67,12 +90,20 @@ class AppGenerator extends Generator {
     writing() {
         this.log('3. Spawn initial project files ...');
         this._copyStaticFiles('static');
```

Comment:
It seems like there are both types of templates copied currently.
We should have a look at the templates in the different "static" folders.